api/prover.proto

```diff
@@ -15,27 +18,7 @@ service Prover {
 }
 
 message EmptyMessage {
-  // Nothing
-}
-
-message PackedDigest {
-  string part1 = 1; // First 253bits of the digest
-  string part2 = 2; // Last 3bits of the digest
-}
-
-message ZethNote {
-  string aPK = 1;
-  string value = 2; // Hexadecimal string representing a int64
-  string rho = 3;
-  string trapR = 4;
-}
-
-message JSInput {
-  repeated string merkleNode = 1; // Merkle path to the commitment
-  int64 address = 2;
-  ZethNote note = 3;
-  string spendingASK = 4;
-  string nullifier = 5;
+  //nothing  
```

Comment:
We can keep it that way. Otherwise, there is an "empty" message defined in https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/empty.proto

Reply:
Fixed